spec/macros/perform_raw_requests.rb

```diff
@@ -0,0 +1,50 @@
+module Macros
+  def it_should_perform_raw_requests
```

Comment:
Assignment Branch Condition size for it_should_perform_raw_requests is too high. [58.08/25]